android/app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +25,21 @@
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
 
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+
+                <data
+                    android:scheme="https"
+                    android:host="app.mutinywallet.com"
+                    android:pathPattern="/.*" />
+                <data
+                    android:scheme="https"
+                    android:host="app.mutinywallet.com" />
+            </intent-filter>
```

Comment:
Figured it out. I believe the data needs to be separate for it to work, additionally on my Android copying or opening any link like [app.mutinywallet.com](http://app.mutinywallet.com) adds `http://` not `https://` in front in the clipboard. I suggest we try something like this.

```suggestion
<intent-filter>
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />

                <data
                    android:scheme="https"
                    android:host="app.mutinywallet.com" />
            </intent-filter>

            <intent-filter>
                <action android:name="android.intent.action.VIEW" />

                <category android:name="android.intent.category.DEFAULT" />
                <category android:name="android.intent.category.BROWSABLE" />

                <data
                    android:scheme="http"
                    android:host="app.mutinywallet.com" />
            </intent-filter>
```